src/sympc/protocol/fss/fss.py

```diff
@@ -124,7 +126,11 @@ def fss_op(x1: MPCTensor, x2: MPCTensor, op="eq") -> MPCTensor:
     Returns:
         MPCTensor: Shares of the comparison.
     """
-    assert not th.cuda.is_available()  # nosec
+    if th.cuda.is_available():
```

Comment:
MAYBE: Save the value of `os.environ["CUDA_VISIBLE_DEVICES"]` and put it back at the end of the ```fss_op```

Reply:
Fixed!